test/functional/test_site/expected/testImportVariables.html

```diff
@@ -62,6 +62,19 @@ <h2 id="trying-to-access-an-imported-variable-via-namespace">Trying to access an
 <script>
   alert("hello")
 </script>
+
+<!-- Global site tag (gtag.js) - Google Analytics -->
+<script async src="https://www.googletagmanager.com/gtag/js?id=TRACKING-ID"></script>
+<script>
+  window.dataLayer = window.dataLayer || [];
+
+  function gtag() {
+    dataLayer.push(arguments);
+  }
+  gtag('js', new Date());
+
+  gtag('config', 'TRACKING-ID');
+</script>
```

Comment:
These changes seem to be unrelated to the PR, I wonder how come our tests didn't fail. :O

Reply:
Shouldn't this be added as part of previous PR 🤔. Why didnt previous PR fail if it does not have this.

Reply:
I think it is because another separate PR was also merged together with your previous PR, and the separate PR added in some new files. Never mind, let's put the fix in this PR, but as a standalone commit.